spec/cobrato/resources/charge_spec.rb

```diff
@@ -9,7 +9,7 @@
       "due_date" => "2015-02-14",
       "document_kind" => "DV",
       "document_date" => nil,
-      "document_number" => nil,
+      "document_number" => 'NA',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.